src/lib/drishti/CMakeLists.txt

```diff
@@ -328,6 +328,9 @@ endif()
 
 foreach(library ${drishti_libs} drishti drishtisdk)
   set_property(TARGET ${library} PROPERTY FOLDER "libs/drishti")
+  if(${DRISHTI_ADD_TO_STRING})
+    target_compile_definitions(${library} PRIVATE DRISHTI_ADD_TO_STRING=1)
```

Comment:
I think we can make it `PUBLIC` and remove other `target_compile_definitions`.

Reply:
I think it can be made `PUBLIC` for `${drishti_libs}` and `PRIVATE` for `drishtisdk`.  That should avoid leaking the definition to end users for the installed lib but reduce some duplication for internal libs.

Reply:
> avoid leaking the definition to end users for the installed lib

May be `BUILD_INTERFACE` is better suitable then?

```cmake
target_compile_definitions(... PUBLIC $<BUILD_INTERFACE:DRISHTI_ADD_TO_STRING>)
```